DigitalRainGame.cs

```diff
@@ -45,8 +73,59 @@ protected override void Draw(GameTime gameTime)
             GraphicsDevice.Clear(Color.Black);
 
             // TODO: Add your drawing code here
+            _spriteBatch.Begin(SpriteSortMode.Deferred, BlendState.NonPremultiplied, SamplerState.PointClamp);
+
+            foreach (IRaindrop raindrop in _raindrops)
+            {
+                raindrop.Draw(_spriteBatch, _font);
+            }
+            _spriteBatch.End();
 
             base.Draw(gameTime);
         }
+
+        //TODO: Test code don't forget to remove this!
+        private void TestSamplerState()
```

Comment:
Was testing out different sprite batch `SamplerState` because they make the fonts look different. Call this function from `Draw()` to see the effects on screen.

These effects are more pronounced because I changed the font size to 16 pixels, (see `Content/Fonts/default_font.spritefont` Line 20). The default size is 12px.